persistence/src/commonMain/db_user/com/wire/kalium/persistence/Members.sq

```diff
@@ -52,7 +52,13 @@ WHERE conversation IN (
 isUserMember:
 SELECT user FROM Member WHERE conversation = :conversationId AND user = :userId;
 
-getMembersFromOneOfTwoDomains:
-SELECT mem.*, con.type FROM Member AS mem
+getFederatedMembersWithOneOfDomainsFromGroupConversation:
```

Comment:
Not relevant, I think we don't have standard, but I see more usages like `select` instead of `get` but not comment on my side